se.kth.iv1350.integration.DatabaseFailureException: DatabaseFailure. Caused by item: 666
	at se.kth.iv1350.integration.ItemRegistry.getItem(ItemRegistry.java:67)
	at se.kth.iv1350.controller.Controller.scanItem(Controller.java:63)
	at se.kth.iv1350.view.View.loopThroughInput(View.java:115)
	at se.kth.iv1350.view.View.sampleExcecution(View.java:95)
	at se.kth.iv1350.startup.Main.main(Main.java:35)
     This occurred at: 2018-05-20T17:58:36.197
se.kth.iv1350.integration.DatabaseFailureException: DatabaseFailure. Caused by item: 666
	at se.kth.iv1350.integration.ItemRegistry.getItem(ItemRegistry.java:67)
	at se.kth.iv1350.controller.Controller.scanItem(Controller.java:63)
	at se.kth.iv1350.view.View.loopThroughInput(View.java:115)
	at se.kth.iv1350.view.View.sampleExcecution(View.java:95)
	at se.kth.iv1350.startup.Main.main(Main.java:35)
     This occurred at: 2018-05-20T21:47:05.218
se.kth.iv1350.integration.DatabaseFailureException: DatabaseFailure. Caused by item: 666
	at se.kth.iv1350.integration.ItemRegistry.getItem(ItemRegistry.java:67)
	at se.kth.iv1350.controller.Controller.scanItem(Controller.java:63)
	at se.kth.iv1350.view.View.loopThroughInput(View.java:115)
	at se.kth.iv1350.view.View.sampleExcecution(View.java:95)
	at se.kth.iv1350.startup.Main.main(Main.java:35)
     This occurred at: 2018-05-25T14:40:23.046
se.kth.iv1350.integration.DatabaseFailureException: DatabaseFailure. Caused by item: 0
	at se.kth.iv1350.integration.ItemRegistry.getItem(ItemRegistry.java:67)
	at se.kth.iv1350.controller.Controller.scanItem(Controller.java:78)
	at se.kth.iv1350.controller.Controller.scanItem(Controller.java:65)
	at se.kth.iv1350.view.View.loopThroughInput(View.java:115)
	at se.kth.iv1350.view.View.sampleExcecution(View.java:95)
	at se.kth.iv1350.startup.Main.main(Main.java:35)
     This occurred at: 2018-05-25T15:44:57.414
java.lang.NullPointerException
	at se.kth.iv1350.model.SaleInformation.updateSale(SaleInformation.java:38)
	at se.kth.iv1350.model.Sale.sellItem(Sale.java:65)
	at se.kth.iv1350.controller.Controller.scanItem(Controller.java:79)
	at se.kth.iv1350.controller.Controller.scanItem(Controller.java:65)
	at se.kth.iv1350.view.View.loopThroughInput(View.java:115)
	at se.kth.iv1350.view.View.sampleExcecution(View.java:75)
	at se.kth.iv1350.startup.Main.main(Main.java:35)
     This occurred at: 2018-05-25T15:47:11.725
se.kth.iv1350.integration.DatabaseFailureException: DatabaseFailure. Caused by item: 0
	at se.kth.iv1350.integration.ItemRegistry.getItem(ItemRegistry.java:67)
	at se.kth.iv1350.controller.Controller.scanItem(Controller.java:78)
	at se.kth.iv1350.controller.Controller.scanItem(Controller.java:65)
	at se.kth.iv1350.view.View.loopThroughInput(View.java:115)
	at se.kth.iv1350.view.View.sampleExcecution(View.java:95)
	at se.kth.iv1350.startup.Main.main(Main.java:35)
     This occurred at: 2018-05-25T15:50:42.208
se.kth.iv1350.integration.DatabaseFailureException: DatabaseFailure. Caused by item: 666
	at se.kth.iv1350.integration.ItemRegistry.getItem(ItemRegistry.java:67)
	at se.kth.iv1350.controller.Controller.scanItem(Controller.java:78)
	at se.kth.iv1350.controller.Controller.scanItem(Controller.java:65)
	at se.kth.iv1350.view.View.loopThroughInput(View.java:115)
	at se.kth.iv1350.view.View.sampleExcecution(View.java:95)
	at se.kth.iv1350.startup.Main.main(Main.java:35)
     This occurred at: 2018-05-25T15:51:10.369
se.kth.iv1350.integration.DatabaseFailureException: DatabaseFailure. Caused by item: 666
	at se.kth.iv1350.integration.ItemRegistry.getItem(ItemRegistry.java:67)
	at se.kth.iv1350.controller.Controller.scanItem(Controller.java:78)
	at se.kth.iv1350.controller.Controller.scanItem(Controller.java:65)
	at se.kth.iv1350.view.View.loopThroughInput(View.java:115)
	at se.kth.iv1350.view.View.sampleExcecution(View.java:95)
	at se.kth.iv1350.startup.Main.main(Main.java:35)
     This occurred at: 2018-05-25T15:55:24.270
